clusters/build-clusters/02_cluster/openshift-apiserver/README.md

```diff
@@ -0,0 +1,48 @@
+# openshift-ingress-operator
```

Comment:
I think it would be better to put this readme together with the certificates, getting from "Oh some certs don't work" to "Oh I might need to look at the openshift-apiserver folders readme" is probably never going to happen for anyone but you

Reply:
Moved.
Corrected the title.